addons/modules/XEH_preInit.sqf

```diff
@@ -0,0 +1,9 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+PREP(moduleAttack);
```

Comment:
These are not needed - the game will prepare functions declared in `CfgFunctions`